Assets/Tests/Weaver/WeaverTestLocator.cs

```diff
@@ -0,0 +1,23 @@
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+
+public class WeaverTestLocator : ScriptableObject
```

Comment:
This is missing a namespace.

might also be worth adding a doc comment to explain what this does